### PR TITLE
feat: edit profile feature and validators

### PR DESCRIPTION
**Pull request by l3luescreen**
📝✅

### CODE REVIEW

File: frontend/src/pages/profile/components/userForm.tsx [hunk 1]
```diff
@@ -11,27 +16,90 @@ const UserForm: React.FC<Props> = ({ user }) => {
...
+      setErrorMessage('ระบุนามสกุลทีพลีส')
+      setFormError(true)
+    } else {
+      setFormError(false)
+      setErrorMessage('')
+    }
+  }
+
+  const submit = async () => {
     // TODO: send update of the user information
```
Comment (loukhin) [inline frontend/src/pages/profile/components/userForm.tsx]: remove todo t please

File: frontend/src/pages/profile/edit.tsx [hunk 2]
```diff
@@ -1,13 +1,21 @@
...
+import { useQuery } from '@apollo/client'
+
 const EditProfile: React.FC = () => {
   const [user, setUser] = useState<ContactInformation>()
 
-  useState(() => {
+  const { loading, data } = useQuery(GET_CURRENT_ACCOUNT)
+
+  useEffect(() => {
     // TODO: get user information
```
Comment (loukhin) [inline frontend/src/pages/profile/edit.tsx]: remove todo t please

File: frontend/src/pages/profile/index.tsx [hunk 3]
```diff
@@ -1,10 +1,25 @@
...
+import { GET_CURRENT_ACCOUNT } from 'lib/queries'
+import { useQuery } from '@apollo/client'
+
 const Profile: React.FC = () => {
+  const [user, setUser] = useState<ContactInformation>()
+
+  const { loading, data } = useQuery(GET_CURRENT_ACCOUNT)
+
+  useEffect(() => {
+    // TODO: get user information
```
Comment (loukhin) [inline frontend/src/pages/profile/index.tsx]: remove todo t please

**Review by loukhin — COMMENTED** [review]
t pls

**Review by loukhin — APPROVED** [review]
🦆

**Review by tanssw — APPROVED** [review]

**Pull request merged**